sequencer/batchbuilder.go

```diff
@@ -271,8 +270,8 @@ func (s *Sequencer) newSequence(ctx context.Context) (types.Sequence, error) {
 	if err != nil {
 		if rollbackErr := dbTx.Rollback(ctx); rollbackErr != nil {
 			return types.Sequence{}, fmt.Errorf(
-				"failed to rollback dbTx when closing batch that gave err: %s. Rollback err: %s",
-				rollbackErr.Error(), err.Error(),
+				"failed to rollback dbTx when closing batch that gave err: %s. Rollback err: %w",
+				rollbackErr.Error(), err,
```

Comment:
```suggestion
				"failed to rollback dbTx when closing batch that gave err: %w. Rollback err: %w",
				rollbackErr, err,
```

Reply:
You can't wrap 2 errors you can only wrap 1 error. Isn't the `err` the one we use in the check later on?